### DISCUSSION TITLE
Daily 22/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#94] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer iba a empezar con la activity de login pero me di cuenta que todavia hay cosa que no se hacer. Estuve haciendo los cuestionarios de entorno y herramientas que tenía pendientes y leyendo un poco de los apuntes. Por la tarde participé de la clase sincrónica. Se agregó un requerimiento de testing para este sprint, así que cree la página en la wiki y realicé el cuadro para agregarle luego el contenido.
Hoy voy a empezar con la activity y por la tarde nos juntamos con algunos del equipo para evacuar dudas.

**Comment by BraianTroncoso** [comment]
Ayer estuve realizando lo que habia dicho. Estuve viendo crud y la mayoria son con un servidor ya sea en spring, node, django, etc. Hoy me pongo a ver el tema de la navegacion.
